src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<>();
+    
+    public static String makeDone(String line) {
+        String msg = "";
+        // sets a specified task as done
+        int index = Integer.valueOf(line.split(" ")[1]);
+        if (index > list.size()) {
+            msg += "No such task!";
+        } else if (list.get(index - 1).isDone) {
```

Comment:
Perhaps you can store what you have get from the list into a new object, then call the method isDone so that it can be more clear.

Example:
Task task = list.get(index);
if (task.isDone) {
//do something
}